includes/admin/payments/orders.php

```diff
@@ -275,7 +275,9 @@ function edd_order_details_customer( $order ) {
 	<?php
 
 	// The edd_payment_personal_details_list hook is left here for backwards compatibility
-	do_action( 'edd_payment_personal_details_list', $payment->get_meta(), $user_info );
+	if ( ! edd_is_add_order_page() ) {
```

Comment:
Just for sanity, can we add an additional check if `$payment` is an object?

Reply:
@ashleyfae since `edd_get_payment` returns either a payment object or `false` is a `! $payment` check sufficient?

Reply:
@robincornett I personally really like to use `instanceof` in cases like this. So:

`&& $payment instanceof EDD_Payment`

That will confirm it's an object and of the expected class, which also means we'll know the `get_meta()` method exists too.